src/submission/submission-entities.ts

```diff
@@ -110,3 +110,22 @@ export interface CreateRegistrationResult {
 export interface ValidationResult {
   errors: DeepReadonly<RegistrationValidationError[]>;
 }
+
+export interface SaveClinicalCommand {
+  records: ReadonlyArray<Readonly<{ [key: string]: string }>>;
+  readonly programId: string;
+  readonly clinicalType: string;
+}
+
+export interface CreateClinicalResult {
+  readonly clinicalData: undefined;
+  readonly successful: boolean;
+  errors: DeepReadonly<ClinicalValidationError[]>;
+}
+
+export type ClinicalValidationError = {
```

Comment:
why not merge this with `RegistrationValidationError` (genralize that one to fit both)